### ISSUE TITLE
Can't disable some mods

### ISSUE THREAD

**Issue by chikachika1** [IllusionMods/KKManager#136]
KK manager version now:  v0.22.0.1
KK manager version before the error: v0.22.0

HF patch version now: v3.22
HF patch version before: v3.21

I tested with both version of KK manager, but without success. Before the last update of HF Patch, I was capable of disable without any issues. 

I read the entire kk manager's log and there's nothing wrong showing up. 

Some dlls are being renamed when disabled, some not. Like I wrote on my another post, I tested and I'm having problem with RealPOV.Koikatu.dll, KK_Ahegao.dll, KK_Pregnancy.dll and KK_Pushup.dll. I tested with another plugins, and these ones are being renamed normally to ".dl_". 

 Later I'll manually rename the plugins to "PLUGIN.dl_" and see what happens.

**Comment by ManlyMarco** [comment]
Do you maybe have multiple copies of said dlls somewhere inside your BepInEx folder?

**Comment by chikachika1** [comment]
> Do you maybe have multiple copies of said dlls somewhere inside your BepInEx folder?

Nope

**Comment by chikachika1** [comment]
I renamed the dlls manually and they came back to "dll"

**Comment by ManlyMarco** [comment]
@chikachika1 Do you have KK_SFW maybe? Try disabling that if you do.

**Comment by chikachika1** [comment]
> @chikachika1 Do you have KK_SFW maybe? Try disabling that in KKManager if you do.

No, the NSFW content is enabled. OBS: I solved the problem renaming the dlls to another extension(Or removing from the folder). Something it's renaming the files from ".dl_" back to ".dll",  but this thing can't re-enable the plugins if you change the file extension to another thing besides ".dl_".

**Comment by chikachika1** [comment]
Should I close the thread or do you want to discover what's causing the bug?

**Comment by ManlyMarco** [comment]
> No, the NSFW content is enabled.

You HAVE to disable the plugin in KKManager or it won't do anything to fix the issue.

**Comment by chikachika1** [comment]
> > No, the NSFW content is enabled.
> 
> You HAVE to disable the plugin in KKManager, changing the setting won't help.

Sure, thank you
